bin/ssh_server.jl

```diff
@@ -11,7 +11,7 @@ sock = listen(port)
 pubkey_to_user_map = Dict{Vector{UInt8}, String}()
 user_to_pubkey_map = Dict{String, Vector{UInt8}}()
 
-const github_token = strip(readstring(joinpath(dirname(@__FILE__),"..","etc","github_tok")))
+const github_token = strip(String(read(joinpath(dirname(@__FILE__),"..","etc","github_tok"))))
```

Comment:
I think the correct replacement is `read(..., String)`

Reply:
I don't think there's anything wrong w/ `String(read(file))` though?

Reply:
I suppose not, but I believe `read(file, String)` avoids creating the intermediate array.

Reply:
I think it creates a `StringVector` underneath, which is essentially the exact same thing as `String(read(file))`